src/ui/public/agg_response/tabify/__tests__/_response_writer.js

```diff
@@ -51,9 +51,9 @@ describe('ResponseWriter class', function () {
 
     it('collects the aggConfigs from each column in aggStack', function () {
       let aggs = [
-        { type: 'date_histogram', schema: 'segment', params: { field: '@timestamp' } },
+        { type: 'date_histogram', schema: 'segment', params: { field: 'time' } },
         { type: 'terms', schema: 'segment', params: { field: 'extension' } },
-        { type: 'avg', schema: 'metric', params: { field: '@timestamp' } }
+        { type: 'avg', schema: 'metric', params: { field: 'bytes' } }
```

Comment:
ugh, I guess I miscalculated this and it was just the type that was an issue (dates in the `avg` agg)